docker-compose.yml

```diff
@@ -44,13 +44,15 @@ services:
     environment:
       DOCKER_HOST_IP: ${DOCKER_HOST_IP}
       ZK: ${DOCKER_HOST_IP}:2181
-      ODE_DDS_CAS_USERNAME: ${ODE_DDS_CAS_USERNAME}
-      ODE_DDS_CAS_PASSWORD: ${ODE_DDS_CAS_PASSWORD}
       ODE_SECURITY_SVCS_SIGNATURE_URI: ${ODE_SECURITY_SVCS_SIGNATURE_URI}
       ODE_RSU_USERNAME: ${ODE_RSU_USERNAME}
       ODE_RSU_PASSWORD: ${ODE_RSU_PASSWORD}
-# Commented out for latest schemaVersion. Uncomment to set for older schemaVersion
-#      ODE_OUTPUT_SCHEMA_VERSION: ${ODE_OUTPUT_SCHEMA_VERSION}
+      ODE_DDS_CAS_USERNAME: ${ODE_DDS_CAS_USERNAME}
+      ODE_DDS_CAS_PASSWORD: ${ODE_DDS_CAS_PASSWORD}
+      # Commented out, will use SDW depositor module by default
+      #        ODE_DEPOSIT_SDW_MESSAGES_OVER_WEBSOCKET: ${ODE_DEPOSIT_SDW_MESSAGES_OVER_WEBSOCKET}
+      # Commented out for latest schemaVersion. Uncomment to set for older schemaVersion
```

Comment:
We don't use `ODE_OUTPUT_SCHEMA_VERSION` any more. We can remove it.

Reply:
Done